packages/typegpu/src/errors.ts

```diff
@@ -86,9 +86,9 @@ export class NotStorageError extends Error {
 }
 
 export class MissingLinksError extends Error {
-  constructor(fnLabel: string | undefined, externalNames: string[]) {
+  constructor(externalNames: string[]) {
     super(
-      `The function '${fnLabel ?? '<unnamed>'}' is missing links to the following external values: ${externalNames}.`,
+      `A function is missing links to the following external values: ${externalNames}.`,
```

Comment:
I would keep passing the label to make it clear which function is missing externals in $uses